packages/lab/src/badge/Badge.css

```diff
@@ -24,44 +6,35 @@
   position: relative;
   flex-shrink: 0;
   vertical-align: middle;
-  line-height: var(--salt-text-lineHeight);
-  font-size: var(--salt-text-fontSize);
-}
-
-/* Give automatic margin when they sit next to each other */
-.saltBadge + .saltBadge {
-  margin-left: var(--badge-sequence-gap);
-}
 
-/* Style applied to the badge in the corner of the node */
-.saltBadge-badge {
-  color: var(--badge-content-text-color);
-  display: inline-block;
-  font-weight: 700; /* TODO: Use characteristic once aligned */
-  font-size: var(--badge-fontSize);
-  /* default small type */
-  width: auto;
-  min-width: 15px;
-  line-height: var(--badge-content-lineHeight);
-  border-radius: var(--badge-content-borderRadius);
-  text-align: center;
+  /* Should this vary according to touch size */
+  padding: 0px var(--salt-spacing-50);
+  line-height: var(--salt-text-lineHeight);
+  min-width: 10px;
+  line-height: 15px;
+  border-radius: 16px;
 
-  transform: translate(var(--badge-offset-inline), -50%);
-  transform-origin: 100% 0%;
-  /*
-    We want offset the next inline element to be strictly next to the badge.
-    This negative margin makes sure this node "appears" to be narrower.
-   */
-  margin-right: var(--badge-offset-inline);
+  font-size: var(--saltBadge-fontSize, var(--salt-text-label-fontSize));
+  font-weight: var(--salt-accent-fontSize);
 
-  padding: var(--badge-content-padding);
-  background: var(--badge-content-icon-color);
+  background: var(--saltBadge-content-icon-color, var(--salt-accent-background));
+  color: var(--saltBadge-content-text-color, var(--salt-accent-foreground));
+}
 
+.saltBadge-inline {
   position: relative;
-
-  flex-wrap: wrap;
   align-items: center;
-  align-content: center;
-  flex-direction: row;
   justify-content: center;
+  display: inline-flex;
+  margin-bottom: 4px;
+  margin-left: var(--salt-spacing-50);
```

Comment:
I would not have margin on badge itself. When it is displayed "inline", the parent should be responsible for its positioning. So not a concern of the badge component.